docs/v0.5/en/contributor-guide/datanode/python-scripts.md

```diff
@@ -1,15 +1,18 @@
-# Python Coprocessor
+# Python Scripts
 
 ## Introduction
 
-Coprocessor, as in TiDB and HBase or Stored Procedure in SQL, is a method to analyze data in local
-databases and send it to remote node instead of sending all data to remote node, which then lets
-remote do the analysis. This way, a lot of data moving costs are saved. This picture below depicts
-how coprocessor works . The `RecordBatch` (Basically a column in table with type and nullability
-metadata) can come from anywhere in the database, and returned `RecordBatch` can be annotated in
-python grammar to indicate its metadata like type or nullability. Coprocessor will try its best to
-convert returned object to a `RecordBatch` , be it a python list, a `RecordBatch` computed from
-parameters, or a constant(which is extends to the same length of input arguments)
+Python scripts are methods for analyzing data in local databases
+and sending it to a remote node instead of sending all the data to the remote node.
```

Comment:
```suggestion
Python scripts are methods for analyzing data in GreptimeDB, by running it in the database directly instead of fetching all the data from the database and running it locally.
```